dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -652,6 +637,38 @@ private void updateInfo(Workflow oldWorkflow, Workflow newWorkflow) {
 
     }
 
+    /**
+     * Register a Zenodo DOI for the workflow version
+     * @param zenodoClient Client for interacting with Zenodo server
+     * @param workflow    workflow for which DOI is registered
+     * @param workflowVersion workflow version for which DOI is registered
+     * @param dockstoreGA4GHBaseUrlParam The baseURL for GA4GH tools endpoint (e.g. "http://localhost:8080/api/api/ga4gh/v2/tools/")
+     * @param dockstoreUrlParam URL for Dockstore (e.g. https://dockstore.org)
+     * @param user user authenticated to issue a DOI for the workflow
+     * @param entryVersionHelper provides interface for retrieving the files of versions
+     */
+    private void registerZenodoDOIForWorkflow(ApiClient zenodoClient, Workflow workflow,
+            WorkflowVersion workflowVersion, String dockstoreGA4GHBaseUrlParam, String dockstoreUrlParam,
```

Comment:
dockstoreGA4GHBaseUrlParam, String dockstoreUrlParam are members of this class, and EntryVersionHelper is implemented by this class -- you don't need to pass any of the 3 as params.

Reply:
fixed